.gitignore

```diff
@@ -2,4 +2,4 @@
 .cabal-sandbox
 cabal.sandbox.config
 dist/*
-
+*_flymake.hs
```

Comment:
You should put this in your global gitignore (yes that's possible).

(I'll still merge this anyways as it doesn't really do much harm.)